### PR TITLE
added new recipe hifi

### PR DESCRIPTION
**Pull request by anacdf**
Hi! There is a new recipe =)
Thank you for the opportunity to contribute

### CODE REVIEW

File: src/recipes/hi-fi.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            "ingredient": "ice"
+        }
+    ],
+    "directions": [
+        "Fill an Old Fashioned glass with ice.",
+        "Pour in vodka.",
+        "Top up with orange soda.",
+        "Garnish with half orange wheel."
+    ],
+    "image": "Hi-Fi.jpg",
```
Comment (alfg) [inline src/recipes/hi-fi.json]: Minor request: can you use all lowercase in the image filename? Thanks!
```suggestion
    "image": "hi-fi.jpg",
```

Comment (alfg) [reply src/recipes/hi-fi.json]: Also 600x400px if possible.

**Comment by anacdf** [comment]
I think it is ok now? =)

**Comment by alfg** [comment]
Hi @anacdf, the filename of the image should be updated as well, please. Thanks!

**Comment by alfg** [comment]
Awesome! Thanks for contributing!

**Pull request merged**